src/map/duel.hpp

```diff
@@ -12,18 +12,23 @@ struct duel {
 	int max_players_limit;
 };
 
-#define MAX_DUEL 1024 //max number of duels on server
-extern struct duel duel_list[MAX_DUEL]; //list of current duel
-extern int duel_count; //current number of duel on server
+//#define MAX_DUEL 1024 //max number of duels on server
+//extern struct duel duel_list[MAX_DUEL]; //list of current duel
+//extern int duel_count; //current number of duel on server
+
+duel& duel_GetDuelId( size_t did );
```

Comment:
duel_getid() will be better, is not it?

Reply:
getbyid then, we don't return the id of the duel but the specific duel from an id. (it's a little confusing since there is the duel module and object)